bloodwork-backend/src/analysis/utils/test-parsing.ts

```
@@ -0,0 +1,552 @@
+          case 4: // Male: MinRef-MaxRef pattern
+            [, testName, , , , unit] = match;
+            value = parseFloat(match[2]);
+            minRef = parseFloat(match[3]);
+            maxRef = parseFloat(match[4]);
+            break;
+            
+          case 5: // Flexible pattern
+            [, testName, , , unit] = match;
+            value = parseFloat(match[2]);
+            const refRange = match[3];
+            if (refRange.includes('-')) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)